elks/arch/i86/drivers/char/console-bios.c

```diff
@@ -41,27 +41,34 @@
 
 #define MAXPARMS        28
 
+#define MAX_OUTPUTS     1
+
 struct console;
 typedef struct console Console;
 
 struct console {
+    unsigned char op;
```

Comment:
Still a byte. I recommend making struct console the exact same (top part at least) between it and console-direct.c.

BTW, console-bios.c is also very useful for checking out various BIOS implementations, I use it a lot in certain debugging situations.